aten/src/ATen/native/CPUBlas.cpp

```diff
@@ -1003,6 +1013,21 @@ struct GemmHelper {
         get_dnnl_dtype(dt_c),
         alpha,
         beta);
+#elif defined(ONEDNN_UKERNEL_2)
+    brg = dnnl::ukernel::brgemm(
+        M,
+        N,
+        K,
+        bs,
+        ld_a,
+        ld_b,
+        ld_c,
+        get_dnnl_dtype(dt_a),
+        get_dnnl_dtype(dt_b),
+        get_dnnl_dtype(dt_c));
+    brg.set_add_C(beta > 0);
```

Comment:
Are alpha and beta not used any more?

Reply:
Yes. oneDNN v3.6 removes the setting of `alpha` and uses `set_add_C` instead of beta.

Reply:
Then you should remove those arguments from `cpublas::brgemm`.

Reply:
To be compatible with oneDNN version 3.5, can we keep these parameters and add checks for beta and alpha in `cpublas::brgemm` for versions >=3.6 ? For example, `TORCH_CHECK(beta == 0 || beta == 1, ...)`, `TORCH_CHECK(alpha == 1, ...)` .

Reply:
Why not just pass `beta = add_C ? 1 : 0` for the old version, then how you can call`cpublas` doesn't depend on the specific version of oneDNN you have installed.

Reply:
Thanks for your comments. I will remove `beta` and `alpha`.

Reply:
Removed.